### PR TITLE
Added db scripts execution on start time

### PR DESCRIPTION
**Pull request by shirmoran**

### CODE REVIEW

**Review by bnshr — APPROVED** [review]
lgtm!

File: k8s/mysql-deployment.yaml [hunk 1]
```diff
@@ -96,7 +144,26 @@ spec:
...
+          postStart:
+            exec:
+              command: 
+                - "/bin/sh"
+                - "-c"
+                - |
+                  while ! mysqladmin ping -uroot -p$MYSQL_ROOT_PASSWORD; do
+                    sleep 10;
+                  done
+                  mysql -uroot -p$MYSQL_ROOT_PASSWORD < /scripts/create_schema.sql
```
Comment (sebrandon1) [inline k8s/mysql-deployment.yaml:159]: Does this need a space between `-p` and the variable?

**Review by sebrandon1 — COMMENTED** [review]
Just one comment about spacing.

File: k8s/mysql-deployment.yaml (hunk 1 above)
Comment (shirmoran) [reply k8s/mysql-deployment.yaml:159]: No, there must not be a space

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**